src/notarytool.ts

```diff
@@ -12,6 +12,12 @@ import { NotaryToolCredentials, NotaryToolStartOptions } from './types';
 
 const d = debug('electron-notarize:notarytool');
 
+function runNotaryTool(args: string[], notarytoolPath?: string) {
+  const useXcrun = typeof notarytoolPath === 'undefined';
```

Comment:
```suggestion
  const useXcrun = notarytoolPath === undefined;
```

No need for `typeof` here, `notarytoolPath` is declared as a parameter.